### ISSUE TITLE
Why latest version is downloading `yolo11n.pt` while training `yolov8*.pt` (or any other model)?

### ISSUE THREAD

**Issue by hdnh2006** [ultralytics/ultralytics#18220]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Ultralytics YOLO Component

Train

### Bug

Hello Ultralytics team,

I noticed that `ultralytics` package is downloading the `yolo11n.pt` model while trying to train any other version of the YOLO.

Below you will find an example with `yolov8s.pt` but the same results occur when training other versions like `yolov9e.pt`, etc. As you can see, the first downloaded model is `yolov8s.pt` but in the next progress bar you will see another download which references to `yolo11n.pt`. 

Maybe there's some mocked code in the latest version by error; I don't know, guys. Can you please check it? This can be confusing for the user..

```bash
yolo train model=yolov8s.pt
Downloading https://github.com/ultralytics/assets/releases/download/v8.3.0/yolov8s.pt to 'yolov8s.pt'...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 21.5M/21.5M [00:00<00:00, 112MB/s]
WARNING ⚠️ 'data' argument is missing. Using default 'data=coco8.yaml'.
Ultralytics 8.3.49 🚀 Python-3.11.0rc1 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 4060 Laptop GPU, 7940MiB)
engine/trainer: task=detect, mode=train, model=yolov8s.pt, data=coco8.yaml, epochs=100, time=None, patience=100, batch=16, imgsz=640, save=True, save_period=-1, cache=False, device=None, workers=8, project=None, name=train, exist_ok=False, pretrained=True, optimizer=auto, verbose=True, seed=0, deterministic=True, single_cls=False, rect=False, cos_lr=False, close_mosaic=10, resume=False, amp=True, fraction=1.0, profile=False, freeze=None, multi_scale=False, overlap_mask=True, mask_ratio=4, dropout=0.0, val=True, split=val, save_json=False, save_hybrid=False, conf=None, iou=0.7, max_det=300, half=False, dnn=False, plots=True, source=None, vid_stride=1, stream_buffer=False, visualize=False, augment=False, agnostic_nms=False, classes=None, retina_masks=False, embed=None, show=False, save_frames=False, save_txt=False, save_conf=False, save_crop=False, show_labels=True, show_conf=True, show_boxes=True, line_width=None, format=torchscript, keras=False, optimize=False, int8=False, dynamic=False, simplify=True, opset=None, workspace=None, nms=False, lr0=0.01, lrf=0.01, momentum=0.937, weight_decay=0.0005, warmup_epochs=3.0, warmup_momentum=0.8, warmup_bias_lr=0.1, box=7.5, cls=0.5, dfl=1.5, pose=12.0, kobj=1.0, nbs=64, hsv_h=0.015, hsv_s=0.7, hsv_v=0.4, degrees=0.0, translate=0.1, scale=0.5, shear=0.0, perspective=0.0, flipud=0.0, fliplr=0.5, bgr=0.0, mosaic=1.0, mixup=0.0, copy_paste=0.0, copy_paste_mode=flip, auto_augment=randaugment, erasing=0.4, crop_fraction=1.0, cfg=None, tracker=botsort.yaml, save_dir=runs/detect/train

                   from  n    params  module                                       arguments                     
  0                  -1  1       928  ultralytics.nn.modules.conv.Conv             [3, 32, 3, 2]                 
  1                  -1  1     18560  ultralytics.nn.modules.conv.Conv             [32, 64, 3, 2]                
  2                  -1  1     29056  ultralytics.nn.modules.block.C2f             [64, 64, 1, True]             
  3                  -1  1     73984  ultralytics.nn.modules.conv.Conv             [64, 128, 3, 2]               
  4                  -1  2    197632  ultralytics.nn.modules.block.C2f             [128, 128, 2, True]           
  5                  -1  1    295424  ultralytics.nn.modules.conv.Conv             [128, 256, 3, 2]              
  6                  -1  2    788480  ultralytics.nn.modules.block.C2f             [256, 256, 2, True]           
  7                  -1  1   1180672  ultralytics.nn.modules.conv.Conv             [256, 512, 3, 2]              
  8                  -1  1   1838080  ultralytics.nn.modules.block.C2f             [512, 512, 1, True]           
  9                  -1  1    656896  ultralytics.nn.modules.block.SPPF            [512, 512, 5]                 
 10                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 11             [-1, 6]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 12                  -1  1    591360  ultralytics.nn.modules.block.C2f             [768, 256, 1]                 
 13                  -1  1         0  torch.nn.modules.upsampling.Upsample         [None, 2, 'nearest']          
 14             [-1, 4]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 15                  -1  1    148224  ultralytics.nn.modules.block.C2f             [384, 128, 1]                 
 16                  -1  1    147712  ultralytics.nn.modules.conv.Conv             [128, 128, 3, 2]              
 17            [-1, 12]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 18                  -1  1    493056  ultralytics.nn.modules.block.C2f             [384, 256, 1]                 
 19                  -1  1    590336  ultralytics.nn.modules.conv.Conv             [256, 256, 3, 2]              
 20             [-1, 9]  1         0  ultralytics.nn.modules.conv.Concat           [1]                           
 21                  -1  1   1969152  ultralytics.nn.modules.block.C2f             [768, 512, 1]                 
 22        [15, 18, 21]  1   2147008  ultralytics.nn.modules.head.Detect           [80, [128, 256, 512]]         
Model summary: 225 layers, 11,166,560 parameters, 11,166,544 gradients, 28.8 GFLOPs

Transferred 355/355 items from pretrained weights
Freezing layer 'model.22.dfl.conv.weight'
AMP: running Automatic Mixed Precision (AMP) checks...
Downloading https://github.com/ultralytics/assets/releases/download/v8.3.0/yolo11n.pt to 'yolo11n.pt'...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5.35M/5.35M [00:00<00:00, 106MB/s]
AMP: checks passed ✅
train: Scanning /home/henry/Projects/CV_solutions/ultralytics/datasets/coco8/labels/train.cache... 4 images, 0 backgrounds, 0 corrupt: 100%|██████████| 4/4 [00:00<?, ?it/s]
val: Scanning /home/henry/Projects/CV_solutions/ultralytics/datasets/coco8/labels/val.cache... 4 images, 0 backgrounds, 0 corrupt: 100%|██████████| 4/4 [00:00<?, ?it/s]
Plotting labels to runs/detect/train/labels.jpg... 
optimizer: 'optimizer=auto' found, ignoring 'lr0=0.01' and 'momentum=0.937' and determining best 'optimizer', 'lr0' and 'momentum' automatically... 
optimizer: AdamW(lr=0.000119, momentum=0.9) with parameter groups 57 weight(decay=0.0), 64 weight(decay=0.0005), 63 bias(decay=0.0)
```

### Environment

```bash
yolo checks
Ultralytics 8.3.49 🚀 Python-3.11.0rc1 torch-2.5.1+cu124 CUDA:0 (NVIDIA GeForce RTX 4060 Laptop GPU, 7940MiB)
Setup complete ✅ (16 CPUs, 14.9 GB RAM, 590.3/937.3 GB disk)

OS                  Linux-6.5.0-45-generic-x86_64-with-glibc2.35
Environment         Linux
Python              3.11.0rc1
Install             pip
RAM                 14.86 GB
Disk                590.3/937.3 GB
CPU                 AMD Ryzen 7 7735HS with Radeon Graphics
CPU count           16
GPU                 NVIDIA GeForce RTX 4060 Laptop GPU, 7940MiB
GPU count           1
CUDA                12.4

numpy               ✅ 2.1.3>=1.23.0
numpy               ✅ 2.1.3<2.0.0; sys_platform == "darwin"
matplotlib          ✅ 3.9.2>=3.3.0
opencv-python       ✅ 4.10.0.84>=4.6.0
pillow              ✅ 11.0.0>=7.1.2
pyyaml              ✅ 6.0.2>=5.3.1
requests            ✅ 2.32.3>=2.23.0
scipy               ✅ 1.14.1>=1.4.1
torch               ✅ 2.5.1>=1.8.0
torch               ✅ 2.5.1!=2.4.0,>=1.8.0; sys_platform == "win32"
torchvision         ✅ 0.20.1>=0.9.0
tqdm                ✅ 4.67.1>=4.64.0
psutil              ✅ 6.1.0
py-cpuinfo          ✅ 9.0.0
pandas              ✅ 2.2.3>=1.1.4
seaborn             ✅ 0.13.2>=0.11.0
ultralytics-thop    ✅ 2.0.12>=2.0.0

```

### Minimal Reproducible Example

```bash
yolo train model=yolov8s.pt
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by UltralyticsAssistant** [comment]
👋 Hello @hdnh2006, thank you for bringing this up and for your detailed issue report! 🚀 

We recommend checking the [Ultralytics Docs](https://docs.ultralytics.com) as a starting point for understanding default behaviors, including model training. You'll find examples for [CLI](https://docs.ultralytics.com/usage/cli/) and [Python](https://docs.ultralytics.com/usage/python/) usage there, which might help clarify your concern.

If this is indeed a 🐛 Bug Report, we appreciate the detailed logs you've provided. However, could you please confirm/reproduce this behavior by ensuring you are running the latest version of the Ultralytics package? Start by upgrading via the following command:

```bash
pip install -U ultralytics
```

We also request a [Minimum Reproducible Example (MRE)](https://docs.ultralytics.com/help/minimum_reproducible_example/) where possible. You've already provided some great context and logs—thank you! If you can identify any additional reproducible steps beyond what you've shared, please add them here.

To ensure the cause is not environment-related, please verify that your setup matches one of our supported environments. YOLO can be run in the following configurations:

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Lastly, our community provides great support options:  
- Join [Discord](https://discord.com/invite/ultralytics) 🎧 to chat in real-time  
- Visit our [Discourse Forum](https://community.ultralytics.com) for detailed discussions  
- Explore our [Subreddit](https://reddit.com/r/Ultralytics) to share ideas  

This is an automated response to help you get started; an Ultralytics engineer will review your issue and provide further assistance soon! 😊 Let's get this sorted out together.

**Comment by Y-T-G** [comment]
Duplicate of https://github.com/ultralytics/ultralytics/issues/17801

**Comment by hdnh2006** [comment]
> Duplicate of #17801

Ok, sorry about it!

**Comment by glenn-jocher** [comment]
@hdnh2006 no problem at all! Let us know if you have any further questions or need assistance.

**Comment by jarvbol** [comment]
> 👋 Привет[@hdnh2006](https://github.com/hdnh2006), спасибо, что подняли этот вопрос и за ваш подробный отчет о проблеме! 🚀
> 
> Мы рекомендуем проверить [Ultralytics Docs](https://docs.ultralytics.com) в качестве отправной точки для понимания поведения по умолчанию, включая обучение модели. Там вы найдете примеры использования [CLI](https://docs.ultralytics.com/usage/cli/) и [Python](https://docs.ultralytics.com/usage/python/) , которые могут помочь прояснить вашу проблему.
> 
> Если это действительно 🐛 Bug Report, мы ценим предоставленные вами подробные журналы. Однако не могли бы вы подтвердить/воспроизвести это поведение, убедившись, что вы используете последнюю версию пакета Ultralytics? Начните с обновления с помощью следующей команды:
> 
> pip install -U ultralytics
> Мы также просим [минимальный воспроизводимый пример (MRE),](https://docs.ultralytics.com/help/minimum_reproducible_example/) где это возможно. Вы уже предоставили отличный контекст и журналы — спасибо! Если вы можете указать какие-либо дополнительные воспроизводимые шаги, помимо тех, которыми вы поделились, пожалуйста, добавьте их здесь.
> 
> Чтобы убедиться, что причина не связана со средой, проверьте, соответствует ли ваша настройка одной из поддерживаемых нами сред. YOLO можно запустить в следующих конфигурациях:
> 
> * **Ноутбуки** с бесплатным GPU:[![Запуск по градиенту](https://camo.githubusercontent.com/6df71b7d7e0b09a2e97776f416bcd40acf48fc87337f8bcce48e4235537daf1f/68747470733a2f2f6173736574732e706170657273706163652e696f2f696d672f6772616469656e742d62616467652e737667)](https://console.paperspace.com/github/ultralytics/ultralytics) [![Открыть в Colab](https://camo.githubusercontent.com/96889048f8a9014fdeba2a891f97150c6aac6e723f5190236b10215a97ed41f3/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667)](https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb) [![Открыть в Kaggle](https://camo.githubusercontent.com/c7135949c5c6882489e68f4af05a78a759460a4db256b86df3097e04419b4d9e/68747470733a2f2f6b6167676c652e636f6d2f7374617469632f696d616765732f6f70656e2d696e2d6b6167676c652e737667)](https://www.kaggle.com/models/ultralytics/yolo11)
> * **Google Cloud** Deep Learning VM. См. [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
> * **Amazon** Deep Learning AMI. См. [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
> * **Образ Docker** . См. [Руководство по быстрому старту Docker.](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) [![Докер тянет](https://camo.githubusercontent.com/6dcbe0a245fadcfd5202d971a92467bd2fcab86efebb2228d73468152c680573/68747470733a2f2f696d672e736869656c64732e696f2f646f636b65722f70756c6c732f756c7472616c79746963732f756c7472616c79746963733f6c6f676f3d646f636b6572)](https://hub.docker.com/r/ultralytics/ultralytics)
> 
> Наконец, наше сообщество предоставляет отличные варианты поддержки:
> 
> * Присоединяйтесь к [Discord](https://discord.com/invite/ultralytics) 🎧, чтобы общаться в режиме реального времени
> * Посетите наш [форум Discourse](https://community.ultralytics.com) для детального обсуждения
> * Исследуйте наш [Subreddit](https://reddit.com/r/Ultralytics) , чтобы поделиться идеями
> 
> Это автоматический ответ, который поможет вам начать работу; инженер Ultralytics рассмотрит вашу проблему и вскоре окажет дополнительную помощь! 😊 Давайте разберемся с этим вместе.

**Comment by twy2020** [comment]
可以找到ultralytics\utils\checks.py，定位到check_amp的amp_allclose，将YOLO("yolo11n.pt")里的路径改为你下载的pt文件的路径，即可完成amp检查

**Comment by glenn-jocher** [comment]
@twy2020 thanks for the suggestion, but modifying the source code isn't recommended as it can cause issues with future updates. The AMP check downloading `yolo11n.pt` is actually expected behavior - it uses a lightweight standard model to test AMP functionality regardless of which model you're training, ensuring consistent and reliable AMP checks across all YOLO versions.
